object-construction-checker/src/main/java/org/checkerframework/checker/objectconstruction/ObjectConstructionAnnotatedTypeFactory.java

```diff
@@ -87,6 +99,9 @@ public void postAnalyze(ControlFlowGraph cfg) {
         getTypeFactoryOfSubchecker(MustCallChecker.class);
     AnnotatedTypeMirror mustCallAnnotatedType =
         mustCallAnnotatedTypeFactory.getAnnotatedType(element);
+    if (mustCallAnnotatedTypeFactory == null) {
```

Comment:
How is this change related to temporary variables?

Reply:
https://github.com/kelloggm/object-construction-checker/pull/291/files/424f947ae09c7edf540ad6c5c9ad821549ab31ed#r570794811